_rules/table-header-cells-refer-to-data-cells-d0f69e.md

```diff
@@ -27,28 +27,20 @@ acknowledgments:
 
 ## Applicability
 
-The rule applies to any HTML element that
+The rule applies to any HTML element with the [semantic role][] of [rowheader][] or [columnheader][] for which all of the following is true:
 
-- has the [semantic role][] of [rowheader][] or [columnheader][] and;
-- is [visible][] and;
-- is [included in the accessibility tree][] and;
-- has at least one ancestor in the [flat tree][] with a [semantic role][] of either [table][] or [grid][]; and
-- whose closest ancestor in the [flat tree][] with a [semantic role][] of either [table][] or [grid][] is [included in the accessibility tree][].
+- the element is [visible][]; and
+- the element is [included in the accessibility tree][]; and
+- the element has at least one ancestor in the [flat tree][] with a [semantic role][] of either [table][] or [grid][]; and
+- the element's closest ancestor in the [flat tree][] with a [semantic role][] of either [table][] or [grid][] is [included in the accessibility tree][].
 
 ## Expectation
 
 Each target element is [assigned][] to at least one element with a [semantic role][] of [cell][] or [gridcell][].
 
-**Note:** The assigned cell is a [descendant][] in the [flat tree][] of the same [table][] or [grid][] element.
-
 ## Assumptions
 
-This rule assumes that table header cells have a relationship conveyed through presentation with other cells within the same table.
-
-**Note:** This assumption helps exclude edge cases like:
-
-- a table definition where there is only one header cell, or
-- a table definition where there are multiple headers and no other cells
+This rule assumes that table header cells have a relationship conveyed through presentation with other cells within the same table. This excludes edge cases such as a table definition where there is only one header cell, or a table definition where there are multiple headers and no other cells. In such scenarios the rule fails, but [success criterion 1.3.1 Info and Relationships][sc1.3.1] could still be satisfied.
```

Comment:
```suggestion
This rule assumes that table header cells have a relationship conveyed through presentation with other table header cells within the same table. This excludes edge cases such as a table definition where there is only one header cell, or a table definition where there are multiple headers and no other cells. In such scenarios the rule fails, but [success criterion 1.3.1 Info and Relationships][sc1.3.1] could still be satisfied.
```

Reply:
@ajanec01 That is incorrect. We assume that there are more than table header cells in the table and that there is a visual relationship from the header cells to non-header cells. If there are zero non-header cells then this rule always fails.

Reply:
Right. The way I read it was that it is assumed that there are multiple table headers and they appear visually next to each other ("...a relationship conveyed through presentation...", "...excludes edge cases such as a table definition where there is only one header cell...").  Sounds good!